sql_generators/glean_usage/common.py

```diff
@@ -40,6 +40,24 @@
     "mozregression",
 )
 
+BIGEYE_APP_EXCLUSION_LIST = (
+    "firefox_echo_show",
+    "firefox_fire_tv",
+    "thunderbird_android",
+    "org_mozilla_social_nightly",
+)
+BIGEYE_APP_EXCLUSION_LIST_METRICS = (
+    "burnham",
+    "firefox_crashreporter",
+    "firefox_fire_tv",
+    "firefox_reality_pc",
+    "mach",
+    "monitor_cirrus",
+    "mozillavpn_backend_cirrus",
+    "mozregression",
+    "thunderbird_desktop",
+)
```

Comment:
Can we move this into the bqetl_project.yaml? https://github.com/mozilla/bigquery-etl/blob/7de5bfbf3a7df2c39ac4b9676cd6a746c528d998/bqetl_project.yaml#L293